ci

```diff
@@ -314,32 +316,32 @@ refresh_hacl_hints () {
   git push $remote $CI_BRANCH
 }
 
-everest_rebuild () {
+function everest_rebuild() {
   git clean -ffdx
-  ./everest --yes $parallel_opt check reset make
+  ./everest --yes $PARALLEL_OPT check reset make
 }
 
-everest_move () {
+function everest_move() {
   # This function is called from a test... so it needs to fast-fail because "set
   # -e" does not apply within subshells.
 
   # VSTS does not clean things properly... no point in fighting that, let's just
   # do it ourselves
   git clean -ffdx
   # Sanity check that will fail if something is off the rails
-  ./everest --yes $parallel_opt check reset || return 1
+  ./everest --yes $PARALLEL_OPT check reset || return 1
   # Update every project to its know good version and branch, then for each
   # project run git pull
   source hashes.sh
   source repositories.sh
-  fresh=false
-  versions=""
+  local fresh=false
+  local versions=""
   for r in ${!hashes[@]}; do
     cd $r
     git pull
     if [[ $(git rev-parse HEAD) != ${hashes[$r]} ]]; then
       fresh=true
-      url=${repositories[$r]#git@github.com:}
+      local url=${repositories[$r]#git@github.com:}
```

Comment:
what is the semantics of declaring a local variable multiple times? this for-loop is going to be executed multiple times